server/controllers/chartsDataController/index.js

```diff
@@ -15,19 +15,21 @@ const show = async (req, res, next) => {
     const { userAccess } = req.session
     const parsedQueryParams = qs.parse(req.query)
     const filters = parsedQueryParams.filters || DEFAULT_CHART_FILTERS
+    const siteList = filters?.sites?.length ? filters.sites : userAccess
```

Comment:
I don't think `filters` will ever not be an object, and we should have a sites key.

```suggestion
    const siteList = filters.sites.length ? filters.sites : userAccess
```

Reply:
[218cfc9](https://github.com/AMP-SCZ/dpdash/pull/441/commits/218cfc99b61b8fbafa89ab15275c64beca1f0b10)